networking_ccloud/ml2/agent/eos/switch.py

```diff
@@ -236,6 +242,72 @@ def get_removable_translations_cmds(iface_name, wanted_translations):
 
         return commands
 
+    def get_config(self):
+        # get infos from the device for everything that we have a model for
+        config = agent_msg.SwitchConfigUpdate(switch_name=self.name, operation=Op.add)
+
+        # vlans
+        for vlan, data in self.send_cmd("show vlan")['vlans'].items():
+            config.add_vlan(int(vlan), data['name'])
+
+        # vxlan_maps
+        vxlan_maps = self.send_cmd("show interfaces Vxlan1")['interfaces']['Vxlan1']['vlanToVniMap']
```

Comment:
`Vxlan1` is hardcoded? O.o

Reply:
`Vxlan1` is a special interface on the `EOS` platform which contains all the vxlan mappings and some other magical stuff. In theory™ there could be multiple Vxlan interfaces, but no EOS platform supports that yet to my knowedge (and our usecase wouldn't need it).